test/helpers/kubectl.go

```diff
@@ -912,37 +912,91 @@ func (kub *Kubectl) CiliumReport(namespace string, commands ...string) {
 	kub.GatherLogs()
 }
 
+// CiliumCheckReport prints a few checks on the Junit output to provide more context to users.
+func (kub *Kubectl) CiliumCheckReport() {
```

Comment:
It might be nice to list the things we're printing here, since the code makes it hard to see because most of it is collecting and formatting.